app/src/main/java/org/mozilla/fenix/trackingprotection/SwitchWithDescription.kt

```diff
@@ -28,19 +29,19 @@ class SwitchWithDescription @JvmOverloads constructor(
                 R.styleable.SwitchWithDescription_switchIcon,
                 R.drawable.ic_tracking_protection
             )
-            switch_widget?.setCompoundDrawablesWithIntrinsicBounds(
-                resources.getDrawable(
-                    id,
-                    context.theme
+            switch_widget.setCompoundDrawablesWithIntrinsicBounds(
```

Comment:
This is unrelated but it'd be nice to use the `putCompoundDrawablesWithIntrinsicBounds` method so we can use named parameters here.

Reply:
Updated!